syncserver/src/server/mod.rs

```diff
@@ -328,10 +334,18 @@ impl Server {
         settings: Settings,
     ) -> Result<dev::Server, ApiError> {
         let settings_copy = settings.clone();
+
         let host = settings.host.clone();
         let port = settings.port;
         let secrets = Arc::new(settings.master_secret.clone());
         let blocking_threadpool = Arc::new(BlockingThreadpool::default());
+        // Adjust the thread count to include FxA blocking threads.
+        let thread_count = settings.worker_max_blocking_threads
+            + settings
```

Comment:
Was this a bug on master that this fixes (we didn't seem to be taking `additional_blocking_threads_for_fxa_requests` into account at all unless `ACTIX_THREADPOOL` wasn't set)?

Reply:
Not really a bug, but more a case that when we the logic worked that if we were running this as a stand-alone TokenServer, we needed to add additional threads to deal with the FxA requests, which are specified in that option. Technically, we could either use just the tokenserver thread count config or the worker_max_blocking_thread count, but I figured we should continue to do the same adjustment.